src/main/java/tech/jhipster/lite/generator/setup/license/domain/LicenseModuleFactory.java

```diff
@@ -0,0 +1,33 @@
+package tech.jhipster.lite.generator.setup.license.domain;
+
+import static tech.jhipster.lite.module.domain.JHipsterModule.*;
+
+import java.time.Year;
+import tech.jhipster.lite.module.domain.JHipsterModule;
+import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+
+public class LicenseModuleFactory {
+
+  public JHipsterModule buildMitModule(JHipsterModuleProperties properties) {
+    //@formatter:off
+    return moduleBuilder(properties)
+      .context()
+        .put("currentYear", Year.now().getValue())
+        .and()
+      .files()
+        .add(from("setup/license/MIT.txt.mustache"), to("./LICENSE.txt"))
+        .and()
+      .build();
+    //@formatter:on
+  }
+
+  public JHipsterModule buildApacheModule(JHipsterModuleProperties properties) {
+    //@formatter:off
+    return moduleBuilder(properties)
+      .files()
+      .add(from("setup/license/Apache.txt"), to("./LICENSE.txt"))
```

Comment:
Following previous comment suggesting to introduce a `SOURCE` constant:

```suggestion
      .add(SOURCE.append("Apache.txt"), to("./LICENSE.txt"))
```